sdk/core/core-util/src/sha256.browser.ts

```diff
@@ -87,8 +82,24 @@ export async function computeSha256Hmac(
   const keyBytes = base64ToBytes(key);
   const stringToSignBytes = utf8ToBytes(stringToSign);
 
-  const cryptoKey = await crypto.importKey("raw", keyBytes, importParams, false, ["sign"]);
-  const signature = await crypto.sign(importParams, cryptoKey, stringToSignBytes);
+  const cryptoKey = await crypto.importKey(
+    "raw",
+    keyBytes,
+    {
```

Comment:
I still cannot believe my eyes. For mysterious reason, when running on React-Native, when I didn't inline the const `importParams`.  Its name got lower-cased!!!

```
### importParams {"name":"hmac","hash":{"name":"SHA-256"}}
```

Reply:
Odd.